deploy/ansible/ballerine_playbook/roles/setup-ballerine/tasks/deploy-ballerine.yml

```diff
@@ -0,0 +1,11 @@
+- name: Deploy Ballerine with localhost
+  shell: sudo docker-compose -f docker-compose-build.yml up -d
+  args:
+    chdir: "{{ install_dir }}/deploy"
+  when: vite_api_url == ""
+
+- name: Deploy Ballerine with custom Domain
+  shell: sudo docker-compose -f docker-compose-build-https.yml up -d
+  args:
+    chdir: "{{ install_dir }}/deploy"
+  when: vite_api_url != ""
```

Comment:
Suggestion: 
Why dont we just manage our monorepo to have additional command which run that command ?
by doing so we can keep maintain those scripts on our main package.json file 

npm run deploy::docker
npm run deploy::docker::custom-domain

Reply:
I wish to use devops tools as it will be easy for any devops engineers to understand how we are building packer image